about.html

```diff
@@ -12,52 +12,105 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
-
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-
-
-    <img src="img/about-working.png" alt="strategy">
-    
-    How We Work
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
 
-    Learn More
-
-
-
-    <img src="img/about-office.png" alt="strategy">
+  <header>
+    <nav>
+        <div class="logo">
+            <img src="img/lambda-black.png"/>
+        </div>
```

Comment:
You already have `<nav>` element specified above, you can target that instead of creating a new container for the links with the class="nav".  Remember that flexbox nests only 2 levels deep so it will not affect the `<a>` elements.